pipeline/article_scraper/processor.py

```diff
@@ -1,15 +1,23 @@
 import pandas as pd
 import os
+import time
+from shutil import copy2
 import sqlalchemy
 from sqlalchemy import create_engine
 from scrapy.crawler import CrawlerProcess
 from scrapy.utils.project import get_project_settings
 
 # Configuration
 data_dir = os.environ['DATA_DIR']
+cfg_dir = os.environ['SCRAPY_CONFIG_DIR']
+cfg_file = cfg_dir + '/scrapy.cfg'
+
 target_data_table = 'RawArticle'
 engine = create_engine('sqlite:///{0}/the_reading_machine.db'.format(data_dir))
 
+# Copy cfg file in the current directory
```

Comment:
The big problem is that the scrapy is looking only in specific location the configuration file.
Or we create our own version of the scrapy handler, otherwise the other trick is to simply copy in one of the folder where it looks for.